docs/compare_collections.md

```diff
@@ -56,15 +56,15 @@ Now that you know the basics, and once you have an understanding of what the com
 
 ## Limitation of the comparison function: distinguishing out-of-order from mismatched arrays
 
-One limitation of the comparison function is that it does comparisons at the level of arrays, not at the level of individual elements. What would the comparison function return for two sequence collections that are exact duplicates except for two component sequences that have swapped names?
+One limitation of the comparison function is that it does comparisons at the level of arrays, not at the level of individual elements. What would the comparison function return for two sequence collections that have the same content, but in different orders, AND where in addition two of the sequences have swapped names?
```

Comment:
```suggestion
One limitation of the comparison function is that it does comparisons at the level of arrays, not at the level of individual elements. What would the comparison function return for two sequence collections that have the same content, but  where two of the sequences have swapped names?
```

Reply:
Another consequence of changing the example (see https://github.com/ga4gh/refget/pull/91/files/fc9624d21aef1486488fe89e5cfe1f3885848f29#r1969833123)